packages/theme/src/windi.ts

```diff
@@ -274,6 +274,13 @@ export const windicssPreset: Config = {
         sora: 'Sora',
       },
 
+      size: {
+        mini: v(ThemeVars.SizeMini),
+        small: v(ThemeVars.SizeSmall),
+        medium: v(ThemeVars.SizeMedium),
+        big: v(ThemeVars.SizeBig),
+      }
+
```

Comment:
That's not the way how Windi works. You should specify sizes for height/widht/font etc explicitly. See: https://windicss.org/utilities/sizing.html